whois-rpsl/src/main/java/net/ripe/db/whois/common/ip/Ipv6Resource.java

```diff
@@ -33,6 +39,18 @@ public class Ipv6Resource extends IpInterval<Ipv6Resource> implements Comparable
     private final long endMsb;
     private final long endLsb;
 
+    private static final Pattern IPV6_SPLIT_PATTERN = Pattern.compile("(" +
+            "[0-9A-Fa-f]{1,4}(?::[0-9A-Fa-f]{1,4}){7}" +
+            "|::" +
+            "|:(?::[0-9A-Fa-f]{1,4}){1,6}" +
```

Comment:
This logic should not be in  this class. This is strictly for IPv6Resource from a string or integer.

Reply:
Created new class